test_programs/compile_success_empty/comptime_type/src/main.nr

```diff
@@ -12,5 +12,22 @@ fn main() {
         let i32_type = type_of(an_i32);
         assert(field_type_1 == field_type_2);
         assert(field_type_1 != i32_type);
+
+        // Check Type::is_field
+        assert(field_type_1.is_field());
+        assert(!i32_type.is_field());
+
+        // Check Type::as_integer
+        assert(field_type_1.as_integer().is_none());
+
+        let (signed, bits) = i32_type.as_integer().unwrap();
+        assert(signed);
+        assert_eq(bits, 32);
+
+        let an_u8: u8 = 0;
+        let u8_type = type_of(an_u8);
```

Comment:
```suggestion
        let a_u8: u8 = 0;
        let u8_type = type_of(a_u8);
```
English is weird. Sounding out "u8," it starts with a "y" sound so the indefinite article "a" is used instead of "an" despite u8 starting with a vowel. Generally the rule is more "starts with a vowel sound" rather than how it is spelled.

Reply:
Sounds good! I think in my head I was thinking of the word "unsigned" but then I would never read that as "unsigned eight"  so not sure what happened in my head 😆